tests/bsim/bluetooth/host/privacy/peripheral/src/tester_rpa_rotation.c

```diff
@@ -56,15 +56,16 @@ static void validate_rpa_addr_generated_for_adv_sets(void)
 			FAIL("RPA not same for adv sets with same id and RPA sharing enabled\n");
 		}
 	}
-	if (bt_addr_le_eq(&adv_set_data[0].old_addr, &adv_set_data[3].old_addr)) {
+	if (bt_addr_le_eq(&adv_set_data[0].old_addr, &adv_set_data[2].old_addr)) {
```

Comment:
Please move the refactors of other tests to a separate PR. You can tag it trivial.
You can keep the rename in this one.

Reply:
Removed the changes